next/utils/networkPages.ts

```diff
@@ -12,9 +15,23 @@ export const getStaticPaths: GetStaticPaths = async () => {
 export const getStaticProps: GetStaticProps<NetworkParam> = async (
   context: GetStaticPropsContext
 ) => {
+  const {
+    dataSource: { url },
+  } = config();
   const { network: networkParam } = context.params ?? {};
 
   const network = NETWORKS.find(({ path }) => path === networkParam) as Network;
+  const projectsResponses = [];
 
-  return { props: { network } };
+  if (network.datasetQueryOrgans.length > 0) {
+    const result = await fetchEntitiesFromQuery(
+      `${url}/projects`,
+      filterSpecimenOrgan(network.datasetQueryOrgans),
+      undefined
+    );
+    const projects = result.hits.map((item) => item.projects[0]);
```

Comment:
Hi @BruceRodrigues would you mind updating the `projectsResponses` type to `ProjectsResponse` from [here](https://github.com/clevercanary/hca-data-portal/blob/875e072d787043cb064666309245369c629f23bc/next/apis/azul/hca-dcp/common/responses.ts).  Update here, and in `NetworkParam` and elsewhere! Currently it is `ProjectResponse[]`. We may need `disease` from `donorOrganisms` from `AggregatedDonorOrganismsResponse` [see here](https://github.com/clevercanary/data-browser/blob/001836735efd803a1e46d02ac432df56e9fb2aef/explorer/app/apis/azul/hca-dcp/common/responses.ts).
Update line `32` to return the `result.hits`.
Thank you!